ts/Grid/Pro/CellEditing/CellEditingComposition.ts

```diff
@@ -302,13 +395,18 @@ declare module '../../Core/Accessibility/A11yOptions' {
 declare module '../../Core/Options' {
     interface ColumnCellOptions {
         /**
-         * Whether to make the column cells editable `true`, or read-only `false`.
-         *
-         * Try it: {@link https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/grid-pro/basic/overview | Editable columns disabled}
-         *
-         * @default true
+         * @deprecated
```

Comment:
Please write a proper PR message with upgrade notes where you explain what has changed.